braille-printer-app/readme.md

```diff
@@ -0,0 +1,131 @@
+Braille Printer Application
```

Comment:
No extra README.md in this sub-directory needed. Merge with the main directory's README.md. Let it have a section for the Printer Application and a section for the classic driver, the Printer Application, being the default/preferred method.